components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -265,8 +265,7 @@ - (void)testCurrentElevationNormal {
   [button setElevation:normalElevation forState:UIControlStateNormal];
 
   // Then
-  XCTAssertEqualWithAccuracy([button elevationForState:button.state],
-                             normalElevation,
+  XCTAssertEqualWithAccuracy([button elevationForState:button.state], normalElevation,
```

Comment:
Per our style guide, if we line-break on one parameter, each parameter should get its own line.